src/transaction/log_prior_send.cpp

```diff
@@ -26,7 +26,7 @@
 namespace cublog
 {
   void
-  prior_sender::send_list (const log_prior_node *head)
+  prior_sender::send_list (const log_prior_node *head, const LOG_LSA *unsent_lsa)
```

Comment:
About the naming of the unsent_lsa, it is hard to understand. How about just catchup_lsa or ps_catchup_lsa? (m_catchup_lsa)